test/functional/CacheManagement/src/tests/sharedclasses/options/TestSharedCacheJavaAPI.java

```diff
@@ -164,14 +164,18 @@ public static void main(String args[]) {
 	       		}
 		       	for(String cacheName: nonpersistentList) {
 	       			if (cacheName.indexOf("groupaccess") != -1) {
-	       				runSimpleJavaProgramWithNonPersistentCache(cacheName, "groupAccess");
+						if (isOpenJ9()) {
```

Comment:
This check affects all platforms, but the original issue complains only about z/OS.